### PR TITLE
Add bare-metal instance as valid

### PR DESCRIPTION
**Pull request by apinnick**
Add "bare-metal instance" to valid list

### CODE REVIEW

**Review by aireilly — APPROVED** [review]
/lgtm

File: .vale/styles/RedHat/TermsErrors.yml [hunk 1]
```diff
@@ -19,8 +19,8 @@ swap:
   "(?<!Mozilla )Thunderbird": Mozilla Thunderbird
   "(?<!pseudo-)ops": operations
   "(?<!self-)hosted engine|hosted-engine": self-hosted engine
-  "bare metal( clusters?| compute| configuration| controls?| environments?| equipment| events?| hardware| hosts?| infrastructure| installations?| installers?| machines?| media| nodes?| provisioning| servers?| workers?)": bare-metal$1
-  "bare-metal(?! clusters?| compute| configuration| controls?| environments?| equipment| events?| hardware| hosts?| infrastructure| installations?| installers?| machines?| media| nodes?| provisioning| servers?| workers?)": bare metal
+  "bare metal( clusters?| compute| configuration| controls?| environments?| equipment| events?| hardware| hosts?| infrastructure| installations?| installers?| instance?| machines?| media| nodes?| provisioning| servers?| workers?)": bare-metal$1
```
Comment (rolfedh) [inline .vale/styles/RedHat/TermsErrors.yml:22]:
```suggestion
  "bare metal( clusters?| compute| configuration| controls?| environments?| equipment| events?| hardware| hosts?| infrastructure| installations?| installers?| instances?| machines?| media| nodes?| provisioning| servers?| workers?)": bare-metal$1
```

File: .vale/styles/RedHat/TermsErrors.yml [hunk 2]
```diff
@@ -19,8 +19,8 @@ swap:
...
+  "bare-metal(?! clusters?| compute| configuration| controls?| environments?| equipment| events?| hardware| hosts?| infrastructure| installations?| installers?| instance?| machines?| media| nodes?| provisioning| servers?| workers?)": bare metal
```
Comment (rolfedh) [inline .vale/styles/RedHat/TermsErrors.yml:23]:
```suggestion
  "bare-metal(?! clusters?| compute| configuration| controls?| environments?| equipment| events?| hardware| hosts?| infrastructure| installations?| installers?| instances?| machines?| media| nodes?| provisioning| servers?| workers?)": bare metal
```

Comment (rolfedh) [reply .vale/styles/RedHat/TermsErrors.yml:23]: Hi @apinnick,

I came across [a Vale issue](https://github.com/redhat-documentation/vale-at-red-hat/issues/871) that might be related to this recent PR you contributed. 
It looks like the ? symbol is used to make the plural s at the end of these strings optional. I've suggested incorporating them here to cover both singular and plural cases. Let me know what you think!

Thanks!

Comment (apinnick) [reply .vale/styles/RedHat/TermsErrors.yml:23]: Aha. OK, now I get it. Your explanation makes sense.

**Pull request merged**